tests/test_interface_parser.py

```diff
@@ -181,29 +181,34 @@ def test_argument_list_templated(self):
     def test_default_arguments(self):
         """Tests any expression that is a valid default argument"""
         args = ArgumentList.rule.parseString(
-            "string c = \"\", string s=\"hello\", int a=3, "
-            "int b, double pi = 3.1415, "
-            "gtsam::KeyFormatter kf = gtsam::DefaultKeyFormatter, "
-            "std::vector<size_t> p = std::vector<size_t>(), "
-            "std::vector<size_t> l = (1, 2, 'name', \"random\", 3.1415)"
+            """string c = "", int z = 0, double z2 = 0.0, bool f = false, """
+            """string s="hello", char c='a', int a=3, """
+            """int b, double pi = 3.1415, """
+            """gtsam::KeyFormatter kf = gtsam::DefaultKeyFormatter, """
+            """std::vector<size_t> p = std::vector<size_t>(), """
+            """std::vector<size_t> l = (1, 2, "name", "random", 3.1415)"""
```

Comment:
Please keep the single and double quote tests. I put those on purpose to test for both cases.

Reply:
done. I had assumed `char c = 'a'` replaced `"name"` but I've also added a char literal inside the tuple.